src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/MemoryLayout.java

```diff
@@ -103,53 +124,30 @@
  * another layout (see {@link MemoryLayout#map(UnaryOperator, PathElement...)}).
  * <p>
  * Such <em>layout paths</em> can be constructed programmatically using the methods in this class.
- * For instance, given a layout constructed as follows:
- * <blockquote><pre>{@code
-SequenceLayout seq = MemoryLayout.ofSequence(5,
-    MemoryLayout.ofStruct(
-        MemoryLayout.ofPaddingBits(32),
-        MemoryLayout.ofValueBits(32, ByteOrder.BIG_ENDIAN).withName("value")
-));
- * }</pre></blockquote>
- *
- * We can obtain the offset, in bits, of the member layout named <code>value</code> from <code>seq</code>, as follows:
+ * For instance, given the {@code taggedValues} layout instance constructed as above, we can obtain the offset,
+ * in bits, of the <em>first</em> member layout named <code>value</code>, as follows:
```

Comment:
Shouldn't it say 'the second member layout'? ('kind' is the first)

Reply:
first in the sequence

Reply:
I've tried to rectified this in the latest revision, see if that works better